react/components/ResourceList/sampleData.js

```diff
@@ -88,126 +58,151 @@ export default {
       email: 'olen.stamm21@yahoo.com',
       name: 'Patrick Rothfuss',
       number: 1.52725,
+      color: badgeData[Math.floor(Math.random() * badgeData.length)],
```

Comment:
Seems a bit cryptic for a color.

Reply:
<img width="491" alt="screen shot 2018-09-25 at 12 16 19" src="https://user-images.githubusercontent.com/8530905/46024271-cb2e6480-c0bc-11e8-9b0a-5c7edc42a3a9.png">